CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta15 (in progress)
+==========================
+
+New Features
+------------
+
+- You can now change the priority of an already published LocalTrack using a new method
+  `setPriority` on the corresponding LocalTrackPublication. (JSDK-2442)
+
+  ```js
+  const localTrackPublication = await room.localParticipant.publishTrack(localTrack, {
+    priority: 'high' // LocalTrack's publish priority - "low", "standard" or "high"
+  });
+
+  // After a while, change the priority to "low".
+  localTrackPublication.setPriority(low);
+  ```
+
+  This will update `publishPriority` on all corresponding RemoteTrackPublications and
+  emit a new event "publishPriorityChanged" to notify the user:
+
+  ```js
+  remoteTrackPublication.on('publishPriorityChanged', priority => {
```

Comment:
This need not be fixed in this change but it looks like the `priority` parameter here is redundant (since its same as  `remoteTrackPublication.publishPriority`). Caller would find it more useful if we passed oldPriority instead of newPriority as a parameter. That way this event would become more easily action-able for the consumer.

Reply:
@makarandp0 Typically, any event that is fired to notify a change in value should have the new value as the parameter in the event handler. The developer usually stores the old value in a separate variable in his/her application logic.

Reply:
maye we should pass both the values then - That could help developers avoid maintaining the old value themselves.